FaceTracking/ViewController.swift

```diff
@@ -228,12 +239,37 @@ extension ViewController: AVCaptureVideoDataOutputSampleBufferDelegate {
 
 extension ViewController {
     func update(with faceRect: CGRect, text: String) {
+        
         DispatchQueue.main.async {
-            UIView.animate(withDuration: 0.2) {
+            UIView.animate(withDuration: 0.2) { [unowned self] in
```

Comment:
in case of animation there is no need to use unowned or weak

Reply:
Thank you. Check.